### ISSUE TITLE
More collectible arrow trails

### ISSUE THREAD

**Issue by qixils** [ProjectEdenGG/Issues#680]
toku requested a pastel rainbow

some other gradient arrows would be cool too

**Comment by GriffinCodes** [comment]
I dont think this was meant to be minigame specific. https://canary.discord.com/channels/132680070480396288/831646358934650920/845768650107387917

**Comment by qixils** [comment]
it wasn't but the issue stands regardless 😛
